test/com/moclojer/resources/moclojer.yml

```diff
@@ -54,3 +54,29 @@
         {
           "project": "{{json-params.project}}"
         }
+
+# This mock register route with multi paths GET /v1/hello/:username
+- endpoint:
+    method: GET
+    path: /v1/hello/:username
+    response:
+      status: 200
+      headers:
+        Content-Type: application/json
+      body: >
```

Comment:
Just to make sure we're not letting this behind:

There're multiples way of defining a multiline string in yaml. Specifically >, |, |> and |+. Could they become a problem? :thinking:

Reply:
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines

for reference.